interface/globals.php

```diff
@@ -465,4 +465,23 @@ function strterm($string,$length) {
   extract($_GET,EXTR_SKIP);
   extract($_POST,EXTR_SKIP);
 }
+//Create page validation array -- all the pages that have to fire validatejs and their rules
+function collectValidationPageRules($option_id,$title){
+    if (!empty($_SESSION['authUserID'])) {
+
+
+        $pages = sqlStatement("SELECT * " .
+            "FROM `list_options` WHERE list_id='PageValidation' AND option_id='$option_id' AND title='$title'");
+
+          $row = sqlFetchArray($pages);
+          if($row) {
+              return array('page_name' => $row['option_id'] . ".php", 'rules' => $row['notes']);
+          }
+          else{
+              return null;
+          }
+
+
+    }
+}
```

Comment:
Do not place this function in globals.php. Instead create a new library script called library/validate_core.php and then require_once the file here:
https://github.com/openemr/openemr/blob/master/interface/globals.php#L176